apps/dashboard/src/routes/org/[slug]/pathways/[id]/settings/+page.svelte

```diff
@@ -0,0 +1,266 @@
+<script lang="ts">
+  import { page } from '$app/stores';
+  import { Restart } from 'carbon-icons-svelte';
+  import {
+    Column,
+    Grid,
+    Row,
+    CodeSnippet,
+    RadioButtonGroup,
+    RadioButton
+  } from 'carbon-components-svelte';
+  import TrashCan from 'carbon-icons-svelte/lib/TrashCan.svelte';
+
+  import { t } from '$lib/utils/functions/translations';
+  import { collection } from '$lib/components/Pathways/store';
+  // future function to handle the saving of collections
+  import { updateCollection } from '$lib/utils/services/pathways';
+  import { currentOrgDomain, currentOrg } from '$lib/utils/store/org';
+
+  import PageNav from '$lib/components/PageNav/index.svelte';
+  import TextArea from '$lib/components/Form/TextArea.svelte';
+  import TextField from '$lib/components/Form/TextField.svelte';
+  import IconButton from '$lib/components/IconButton/index.svelte';
+  import { VARIANTS } from '$lib/components/PrimaryButton/constants';
+  import UploadWidget from '$lib/components/UploadWidget/index.svelte';
+  import PrimaryButton from '$lib/components/PrimaryButton/index.svelte';
+  import { handleOpenWidget } from '$lib/components/CourseLandingPage/store';
+  import CustomToggle from '$lib/components/Pathways/Toggle.svelte';
+
+  let id: string;
+  let isSaving = false;
+  let slug: boolean = true;
+  let errors: {
+    title: string | undefined;
+    description: string | undefined;
+  } = { title: undefined, description: undefined };
+
+  function widgetControl() {
+    $handleOpenWidget.open = !$handleOpenWidget.open;
+  }
+
+  const deleteBannerImage = () => {
+    $collection.avatar = '';
+  };
+
+  const generateNewCourseLink = () => {
+    // console.log('generate course link');
+  };
+
+  const handleSave = async () => {
+    if (!$collection.title) {
```

Comment:
@tunny17 Please review your code naa, I have to recheck and recheck..

You are still using the word `collection`